Content/ReleaseBriefings/4_6/Section_BackendUpdates.tex

```diff
@@ -0,0 +1,38 @@
+%==========================================================================
+
+\begin{frame}[fragile]
+
+  {\Huge Backend Updates}
+
+  \vspace{10pt}
+
+\end{frame}
+
+
+%==========================================================================
+
+% Examples
+
+% note: always keep the [fragile] for your frames!
+
+\begin{frame}[fragile]{CUDA, SYCL and Serial}
+  \begin{itemize}
+      \item CUDA: Improved performance for \texttt{Kokkos::parallel\_reduce} on H100 and newer by increasing launch bounds
+      \item SYCL: Improved sorting performance for non-contiguous views with \texttt{RandomAccessIterator}
```

Comment:
```suggestion
      \item SYCL: Improved sorting performance for non-contiguous views
```
Is that different from the general improvement to the iterators?

Reply:
Yes, we are not falling back to sorting on the host anymore (given a sufficiently new oneDPL version).